docs/app/Components/ComponentDoc/ComponentProps.js

```diff
@@ -3,11 +3,6 @@ import React, { Component, PropTypes } from 'react'
 
 import { Label, Table } from 'src'
 
-const descriptionExtraStyle = {
-  fontSize: '0.95em',
-  color: '#777',
-}
-
```

Comment:
No, I removed it consciously. Enum's functionality relies on `meta.props`, this PR makes it obsolete, we will remove it in future PRs. See [this](https://github.com/Semantic-Org/Semantic-UI-React/pull/731#issuecomment-258778858) comment and `Rail`'s update in this PR.